components/automate-ui/src/app/components/settings-sidebar/settings-sidebar.component.spec.ts

```diff
@@ -110,10 +105,6 @@ describe('SettingsSidebarComponent', () => {
   });
 
   describe('IAM v1', () => {
-    beforeEach(() => {
-      component.iamMajorVersion$ = observableOf(<IAMMajorVersion>'v1');
```

Comment:
Wondering if this doesn't need a corresponding `observableOf(false)` somewhere...?